javascript/javascript1/week1/homework/ez-namey.js

```diff
@@ -0,0 +1,9 @@
+
+
+console.log("======Exercise Ez Namey======");
+
+
+let firstWords = ["Easy", "Awesome", "Corporate", "Quick", "Smart", "Affordable", "Good", "Interesting", "Agile", "Compact"];
+let secondWords = ["Day", "Firm", "Tech", "Corporation", "IT", "Intelligence", "Business", "Exploration", "Solutions", "Company"];
+let startupName = firstWords[Math.floor(Math.random() * 10)] + " " + secondWords[Math.floor(Math.random() * 10)];
+console.log('The startup:', '"'+ startupName +'"', 'contains', startupName.length + ' characters.');
```

Comment:
When you add the `,` you get the `"` quotemarks in the console log message as well, so now it look look like:
```
"The startup:" "'Corporate Day'" "contains" "7 characters."
```
more simple would be if you change the log to:
```suggestion
console.log("The startup: " + "'" + startupName + "'" + " contains " + startupName.length + " characters.");
```
this will print:
```
"The startup: 'Corporate Day' contains 13 characters."
```